test/module/irohad/ametsuchi/ametsuchi_test.cpp

```diff
@@ -750,3 +752,81 @@ TEST_F(AmetsuchiTest, AddSignatoryTest) {
     ASSERT_EQ(signatories->at(0), pubkey1);
   }
 }
+
+TEST_F(AmetsuchiTest, FindTxByHashTest) {
```

Comment:
holy shi~
is it possible to split it in several tests?

Reply:
Well, all ametsuchi_test should be reworked but it should be done as a separate task.